README.md

```diff
@@ -1594,6 +1594,22 @@ Server: spray-can/1.3.3
 }
 ```
 
+Query data is refreshed from raw data periodically according to the configuration value `services.MetadataService.metadata-summary-refresh-interval`.
+If not specified the refresh interval will default to 2 seconds.  
```

Comment:
Can you specify if that's every n seconds, or n seconds after the previous one? 

I was going to ask that question, said I could look further down into the diff, and then figured if I was curious that I might be the only one who wonders that in the future :)